engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaBackendIsCopyingCachedOutputsSpec.scala

```diff
@@ -96,17 +98,17 @@ class EjeaBackendIsCopyingCachedOutputsSpec extends EngineJobExecutionActorSpec
         }
 
         RestartOrExecuteCommandTuples foreach { case RestartOrExecuteCommandTuple(operationName, restarting, expectedMessage) =>
-          s"$operationName the job immediately when it gets a failure result, and it was going to receive $hashComboName, if call caching is $mode" in {
+          s"invalidate a call for caching if backend coping failed when start mode is $operationName, and it was going to receive $hashComboName, if call caching is $mode" in {
             ejea = ejeaInBackendIsCopyingCachedOutputsState(initialHashData, mode, restarting = restarting)
             // Send the response from the copying actor
             ejea ! failureNonRetryableResponse
 
-            helper.bjeaProbe.expectMsg(awaitTimeout, expectedMessage)
-            ejea.stateName should be(RunningJob)
-            ejea.stateData should be(ResponsePendingData(helper.backendJobDescriptor, helper. bjeaProps, initialHashData))
+            expectInvalidateCallCacheActor(cacheId)
+            ejea.stateName should be(InvalidatingCacheEntry)
```

Comment:
Is there a race condition where the `expectInvalidateCallCacheActor(cacheId)` might become true before the ejea changes state?
If so, add `eventually { ... }` around the `ejea.stateName should be(InvalidatingCacheEntry)`